test/scss/_config.scss

```diff
@@ -8,6 +8,18 @@ $fonts: (
     'italic': false,
   ),
 
+  'TestFont': (
+    'stack': ('Georgia', 'serif'),
+    'normal': (
+      'local': true,
```

Comment:
What is this line used for?

Reply:
This was an attempt to turn the local setting on/off for a particular font. If truthy, we add local. If a string or string-list is given, we use that - otherwise we fall back to the name.